lib/src/main/java/net/dean/jraw/models/Gildings.java

```diff
@@ -0,0 +1,28 @@
+package net.dean.jraw.models;
+
+import com.google.auto.value.AutoValue;
+import com.squareup.moshi.Json;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+import java.io.Serializable;
+
+@AutoValue
+public abstract class Gildings implements Serializable {
+
+    /** Number of silver coins of the object */
+    @Json(name = "gid_1")
+    public abstract short getSilvers();
```

Comment:
I can't find any instance where `gid_x` values are ever null. If you can't find one either, these should all be `@NotNull`.

Reply:
Neither do I